website/docs/components/form/super-select/partials/specifications/anatomy.md

```diff
@@ -0,0 +1,17 @@
+## Anatomy
+
+![Anatomy of superSelect](/assets/components/form/super-select/super-select-anatomy.png =667x*)
+
+
+<!-- this is just an example, refer to other components to see how to fill this table -->
+| Element          | Usage                                           |
+|------------------|-------------------------------------------------|
+| Label             | Required                                                    |
+| Indicator         | Optional                                                    |
```

Comment:
Looking at our other form components, we do the same thing, but "Indicator" seemed strange to me. If we're going to change it we should do it more broadly, so might be out of the scope of this, but just noting it.

Maybe it should be "Required indicator"? But then you get into this scenario of:

```markdown
| Required indicator | Optional |
```

which is just weird. Not sure of the right solution currently.

Reply:
@majedelass I took a look at some of the other Form component docs and looks like it's just called "Indicator"

https://helios.hashicorp.design/components/form/select?tab=specifications

<img width="869" alt="image" src="https://github.com/hashicorp/design-system/assets/108769823/db69d013-3f80-44c0-a021-8cfee81e7cdf">

Reply:
@majedelass So I think the way you have it is fine. We could look at opening a ticket later on to update this for all the form components at once if it's deemed confusing...